commit d5d12dc56c5ca35c99f03351b390c605db3d1d3e (refs/changes/68/3368/2)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2018-02-14T16:51:51-08:00 (1 year, 8 months ago)
    
    ADMINUI-2398 CN Setup always shows 'Setup Failed' during setup
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/sapi_manifests/adminui/template b/sapi_manifests/adminui/template
index b46d43a2..74a02725 100644
--- a/sapi_manifests/adminui/template
+++ b/sapi_manifests/adminui/template
@@ -77,5 +77,5 @@
 			}
 		}
 	},
-	"serverSetupJobName": "server-setup-1.0.2"
+	"serverSetupJobName": "server-setup-*"
 }
